scripts/entrypoint.sh

```diff
@@ -62,23 +62,31 @@ main () {
     local terraform_success=true
     case "${arg_action}" in
         'create')
-            entrypoint_helpers::create \
+            {
+                entrypoint_helpers::create \
                 "${arg_cluster}" \
                 "${tmp_var_file}" \
-                "${module_path}"
+                "${module_path}" \
+                && echo "Successfully created Cluster...." >&2
+            } \
+            || { echo "Failed to create Cluster...."; return 1;} >&2
```

Comment:
I think this will return from the function before copying the logs and tfvars to the gcs bucket. Is that intended?

Reply:
that is not intended. Fixed in latest iteration.